src/Compute/Compute/Compute.csproj

```diff
@@ -15,6 +15,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.PowerShell.AutoMapper" Version="6.2.2" />
     <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.1.0" />
+    <PackageReference Include="System.IO.FileSystem.AccessControl" Version="5.0.0" />
```

Comment:
Why is this here?

Reply:
Adding AccessControl for File Access Control to reduce permissions for private key